core/src/de/golfgl/gdxgamesvcs/player/IPlayerData.java

```diff
@@ -0,0 +1,44 @@
+package de.golfgl.gdxgamesvcs.player;
+
+/**
+ * Leaderboard entry data
+ *
+ * @author Kari Vatjus-Anttila
+ */
+public interface IPlayerData {
+
+    /**
+     * Player Id
+     *
+     * @return Player Id
+     */
+    public String getPlayerId();
+
+    /**
+     * Player Display Name
+     *
+     * @return Player Display Name
+     */
+    public String getDisplayName();
+
+    /**
+     * Player Name
+     *
+     * @return Player Name
+     */
+    public String getName();
+
+    /**
+     * Player Title
+     *
+     * @return Player Title
+     */
+    public String getTitle();
+
+    /**
+     * Player Retrieved Timestamp
+     *
+     * @return Player Retrieved Timestamp
+     */
+    public long getRetrievedTimestamp();
```

Comment:
what is a retrieved timestamp?

Reply:
From the [docs](https://developers.google.com/android/reference/com/google/android/gms/games/Player#getRetrievedTimestamp())

`Retrieves the timestamp at which this player record was last updated locally.`